test/unit/puppetlabs/services/jruby/jruby_testutils.clj

```diff
@@ -180,6 +130,7 @@
                          tk-service/service-context
                          :pool-context)
         num-jrubies (-> pool-context
+                        :internal
```

Comment:
It probably would be good to change this function to do

```clojure
(-> pool-context
     jruby-core/get-pool-state
    :size)
```

since the `:internal` key is supposed to be private,so that in case we ever change where they pool state is kept, we won't have to change this here.